samples/crypto/psa_tls/README.rst

```diff
@@ -29,6 +29,217 @@ Overview
+
+.. tabs::
+
+   .. tab:: TLS v1.2
+
+      .. list-table::
+         :header-rows: 1
+         :widths: auto
+
+         * - Board target
+           - Backend
+           - CMSE supported
+           - DTLS supported
+           - Limitations / not supported
+           - Comments
+         * - ``nrf52840dk/nrf52840``
+             ``nrf9160dk/nrf9160``
+             ``nrf9151dk/nrf9151``
+           - :ref:`Mbed TLS legacy c33xx backend <nrf_security_backends_cc3xx>`
+           - No
+           - No
+           - AES256, AES-GCM, SHA-512
+           - Also supports :ref:`PSA Crypto nrf_oberon driver<nrf_security_drivers_oberon>`
+         * - ``nrf52840dk/nrf52840``
+             ``nrf9160dk/nrf9160``
+             ``nrf9151dk/nrf9151``
+           - :ref:`Mbed TLS legacy oberon backend<nrf_security_backends_oberon>`
+           - No
+           - No
+           -
+           -
+         * - ``nrf52840dk/nrf52840``
+           - :ref:`PSA Crypto cc3xx driver<nrf_security_drivers_cc3xx>`
+           - No (secure only)
+           - Yes
+           - RSA
+           - Also supports :ref:`PSA Crypto nrf_oberon driver<nrf_security_drivers_oberon>`
+         * - ``nrf9160dk/nrf9160``
+             ``nrf9151dk/nrf9151``
+           - :ref:`PSA Crypto cc3xx driver<nrf_security_drivers_cc3xx>`
+           - Yes
+           - Yes
+           - RSA
+           - Also supports :ref:`PSA Crypto nrf_oberon driver<nrf_security_drivers_oberon>`
+         * - ``nrf5340dk/nrf5340/cpuapp``
+           - :ref:`Mbed TLS legacy c33xx backend <nrf_security_backends_cc3xx>`
+           - No
+           - No
+           -
+           - Also supports :ref:`PSA Crypto nrf_oberon driver<nrf_security_drivers_oberon>`
+         * - ``nrf5340dk/nrf5340/cpuapp``
+           - :ref:`Mbed TLS legacy oberon backend<nrf_security_backends_oberon>`
+           - No
+           - No
+           -
+           -
+         * - ``nrf5340dk/nrf5340/cpuapp``
+           - :ref:`PSA Crypto cc3xx driver<nrf_security_drivers_cc3xx>`
+           - Yes
+           - Yes
+           - RSA
+           - Also supports :ref:`PSA Crypto nrf_oberon driver<nrf_security_drivers_oberon>`
+         * - ``nrf54l15dk/nrf54l15/cpuapp``
+           - :ref:`PSA Crypto CRACEN driver<nrf_security_drivers_cracen>`
+           - Yes
+           - Yes
+           -
+           - Also supports :ref:`PSA Crypto nrf_oberon driver<nrf_security_drivers_oberon>`
+         * - ``nrf54l15dk/nrf54l10/cpuapp``
+           - :ref:`PSA Crypto CRACEN driver<nrf_security_drivers_cracen>`
+           - No
+           - Yes
+           -
+           - Also supports :ref:`PSA Crypto nrf_oberon driver<nrf_security_drivers_oberon>`
+         * - ``nrf54h20dk/nrf54h20/cpuapp``
+             ``nrf54h20dk/nrf54h20/cpurad``
+           - `SSF Crypto <CONFIG_SSF_PSA_CRYPTO_SERVICE_ENABLED_>`_
+           - No
+           - Yes
+           -
+           - Also supports :ref:`PSA Crypto nrf_oberon driver<nrf_security_drivers_oberon>`
```

Comment:
Leave this as blank as well.
SSF crypto service does not support nrf_oberon driver due to other reasons. The explanation might be a bit long for this comment. Happy to have a discussion offline.